graphics/src/ColladaLoader.cc

```diff
@@ -1992,15 +1993,16 @@ void ColladaLoaderPrivate::LoadPolylist(tinyxml2::XMLElement *_polylistXml,
 
   std::vector<ignition::math::Vector3d> verts;
   std::vector<ignition::math::Vector3d> norms;
-  std::vector<ignition::math::Vector2d> texcoords;
+  std::map<unsigned int, std::vector<ignition::math::Vector2d>> texcoords;
+  std::map<unsigned int, unsigned int> texcoordsOffsetToSet;
 
   const unsigned int VERTEX = 0;
   const unsigned int NORMAL = 1;
   const unsigned int TEXCOORD = 2;
   unsigned int otherSemantics = TEXCOORD + 1;
 
   // look up table of position/normal/texcoord duplicate indices
-  std::map<unsigned int, unsigned int> texDupMap;
+  std::map<unsigned int, std::map<unsigned int, unsigned int>> texDupMap;
```

Comment:
Do the new maps need to be ordered? I see there's a concept of "first" set for backwards compatibility, but maybe that can be worked around?

Reply:
Nope, I updated the new outer map to be unordered.

about the "first" uv, I was mainly thinking of a use case where some one wrote a mesh loader that can selectively load any uv set in the mesh, which end up with code like:

e.g. 
```
int set = 1
AddTextureCoordinateBySet(u0, v0, set)
AddTextureCoordinateBySet(u1, v1, set)
...
auto count = TextureCoordinateCount()
```
In this case, I just return the number of texture coordinates that are in the only set in the submesh. We could decide that `count` in the above code will just be 0, and we can get rid of `first*`. I'm open two either option.

Reply:
I don't have a strong opinion here... I'd also be ok returning the first element of the unordered map, which may be undefined if there are multiple sets... Maybe we can print a warning if they use this function and there's more than one set? I'm fine either way.

Reply:
ok I tried the simple case of just defaulting the first set to index 0u but ran into a test failure due to a very similar scenario described in my previous comment, i.e. there was a mesh with only one tex coord set with index=1. I ran into another test failure when changing to use `unordered_map` in SubMesh as first tex coord set (indicated by `begin()`) changed when a new set is added. In the end I just added warning messages for the case where there are multiple uv sets in the submesh. bcdac31